config/config.go

```diff
@@ -15,28 +15,41 @@ import (
 var Config *config
 
 type config struct {
-	IPFS               ipfs          `yaml:"ipfs"`
-	PublicHLSPath      string        `yaml:"publicHLSPath"`
-	PrivateHLSPath     string        `yaml:"privateHLSPath"`
-	VideoSettings      videoSettings `yaml:"videoSettings"`
-	Files              files         `yaml:"files"`
-	FFMpegPath         string        `yaml:"ffmpegPath"`
-	WebServerPort      int           `yaml:"webServerPort"`
-	S3                 s3            `yaml:"s3"`
-	EnableOfflineImage bool          `yaml:"enableOfflineImage"`
+	IPFS           ipfs          `yaml:"ipfs"`
+	PublicHLSPath  string        `yaml:"publicHLSPath"`
+	PrivateHLSPath string        `yaml:"privateHLSPath"`
+	VideoSettings  videoSettings `yaml:"videoSettings"`
+	Files          files         `yaml:"files"`
+	FFMpegPath     string        `yaml:"ffmpegPath"`
+	WebServerPort  int           `yaml:"webServerPort"`
+	S3             s3            `yaml:"s3"`
 }
 
 type videoSettings struct {
 	ChunkLengthInSeconds int             `yaml:"chunkLengthInSeconds"`
 	StreamingKey         string          `yaml:"streamingKey"`
-	EncoderPreset        string          `yaml:"encoderPreset"`
-	StreamQualities      []streamQuality `yaml:"streamQualities"`
+	StreamQualities      []StreamQuality `yaml:"streamQualities"`
+	OfflineContent       string          `yaml:"offlineContent"`
 	EnablePassthrough    bool            `yaml:"passthrough"`
-	OfflineImage         string          `yaml:"offlineImage"`
 }
 
-type streamQuality struct {
-	Bitrate int `yaml:"bitrate"`
+type StreamQuality struct {
+	// Enable passthrough to copy the video and/or audio directly from the
+	// incoming stream and disable any transcoding.  It will ignore any of
+	// the below settings.
+	IsVideoPassthrough bool `yaml:"videoPassthrough"`
+	IsAudioPassthrough bool `yaml:"audioPassthrough"`
+
+	Bitrate      int `yaml:"bitrate"`
```

Comment:
I left this as `bitrate` instead of `videoBitrate` because the config files already use `bitrate`, though I should probably just rename it.

Reply:
Yeah, renaming this to have the `video` prefix would probably be user friendly even though there is `audioBitrate`.